src/gp/src/GPMSA.C

```diff
@@ -766,7 +767,8 @@ GPMSAFactory<V, M>::setUpEmulator()
 
   for (unsigned int i=0; i != m_numSimulations; ++i)
     for (unsigned int j=0; j != numOutputs; ++j)
-      (*simulationOutputMeans)[j] += (*m_simulationOutputs[i])[j];
+      (*simulationOutputMeans)[j] +=
```

Comment:
This code is correct, but it's subtle.  If you're doing mean/variance normalization on a single index i, then simulationOutputMeans will be 0, and calculating it is a waste of time, but still gives the correct result.

But, if you're normalizing based on min/max, or a user-submitted range, or if index i is only one index of many in a partition, then the mean of the normalized values will not be zero, and yet we'll still want to subtract it off before doing the SVD.